Makefile

```diff
@@ -0,0 +1,8 @@
+MICROOVN_SNAP_PATH=$(CURDIR)/microovn.snap
+build:
+	@echo "Building the snap"
+	@snapcraft pack --use-lxd -o $MICROOVN_SNAP_PATH
+
+func-tests:
+	@echo "Running functional tests"
+	@bats tests/basic_cluster.bats
```

Comment:
I believe the target provided to `bats` can be a directory so that it will execute all the *.bats file in that directory.